### PR TITLE
Update the README to include play/role summaries

### LINKED ISSUE #44: Update README to document what the rpc-extras repo includes
We should provide a high level overview of what the rpc-extras repo includes. This should help others that implement projects on top of os-ansible-deployment to see how to implement their add-ons.

### PR DESCRIPTION
**Pull request by nrb**
Fixes #44

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,27 @@
 # rpc-extras
 Optional add-ons for Rackspace Private Cloud
+
+# os-ansible-deploy integration
+
+The rpc-extras repo includes add-ons for the Rackspace Private Cloud product
+that integrate with the os-ansible-deploy set of Ansible playbooks and roles.
```
Comment (stevelle) [inline README.md]: link to https://github.com/stackforge/os-ansible-deployment

**Comment by miguelgrinberg** [comment]
:+1:

File: README.md [hunk 2]
```diff
@@ -1,2 +1,30 @@
...
+
+# Ansible Playbooks
+
+* `horizon_extensions.yml` - rebrands the horizon dashboard for Rackspace,
+as well as adding a Rackspace tab and a Solutions tab, which provides
+Heat templates for commonly deployed applications.
+* `rpc-support.yml` - provides holland backup service, support SSH key
+distribution, custom security group rules, bashrc settings, and other
+miscellaneous tasks helpful to support personnel.
+* `setup-maas.yml` - deploys, sets up, and installs Rackspace MaaS checks
```
Comment (stevelle) [inline README.md]: should we add a link to http://www.rackspace.com/cloud/monitoring while we're here?  (yeah, should have caught that last time too.

Comment (nrb) [inline README.md]: Done.

**Comment by stevelle** [comment]
+1

**Pull request merged**